PIMTool.Client/Presentation/ViewModels/DetailProjectViewModel.cs

```diff
@@ -132,51 +181,55 @@ public string Error
         }
         #endregion
 
-        public EditProjectViewModel(MainViewModel mainViewModel, ProjectResource project)
+        public DetailProjectViewModel(MainViewModel mainViewModel, bool isEditMode, ProjectResource project = null)
         {
+            IsEditMode = isEditMode;
             _mainViewModel = mainViewModel;
-            _projectValidator = new EditProjectValidator();
+            _projectValidator = new DetailProjectValidator(_mainViewModel.Projects, isEditMode);
             _projectWebApiClient = _mainViewModel.ProjectWebApiClient;
-            Project = project;
-            LoadProject();
-            UpdateProjectCommand = new UpdateProjectCommand(UpdateProject, _mainViewModel);
+            EditedProject = project;
+            if (IsEditMode)
+            {
+                ProjectNumberEnable = false;
+                ButtonMode = "Update";
+                LoadProject();
+            }
         }
 
-        //TODO: Update Project to the DB
-        private void UpdateProject()
+        //Update Project to the DB
+        private void SaveOrUpdateProject()
         {
             var project = new SaveProjectResource();
-            project.ProjectNumber = (int)ProjectNumber;                                                                                                                                                             
+            project.ProjectNumber = ProjectNumber;
             project.GroupId = Group + 1;
```

Comment:
Why Group = Group + 1, if I edit and save Project n times then GroupId will increase to n?